tagreader.yaml

```diff
@@ -68,40 +70,14 @@ ota:
 
 # Enable SPI interface
 spi:
-  clk_pin: D0
-  miso_pin: D1
-  mosi_pin: D2
-
-# Configure the PN532 module
-pn532:
-  cs_pin: D3
-  update_interval: 2s
-
-  # What happens when a tag is read
-  on_tag:
-    then:
-    - homeassistant.tag_scanned: !lambda 'return x;'
-    - if:
-        condition:
-          switch.is_on: buzzer_enabled
-        then:
-        - rtttl.play: "success:d=24,o=5,b=100:c,g,b"
-    - if:
-        condition:
-          switch.is_on: led_enabled
-        then:
-        - light.turn_on:
-            id: activity_led
-            brightness: 100%
-            red: 0%
-            green: 100%
-            blue: 0%
-            flash_length: 500ms
+  clk_pin: D5
```

Comment:
Can we set these via substitution so it's easier to customize

Reply:
We could yes, maybe as part of another PR then with a bit of tidy up

Reply:
why use substitutions I understand in the case of names that you would have to type several times 
But where is the benefit here?

Reply:
You can keep using the original YAML but still have a different configuration for your board. It's about splitting configuration (which pin is connected to what) and what to do with each pin (the YAML)